docs/core/extensions/httpclient-http3.md

```diff
@@ -18,15 +18,15 @@ HTTP/3 and QUIC have a number of benefits compared to HTTP/1.1 and HTTP/2:
 
 > [!IMPORTANT]
 >
-> HTTP/3 is available in .NET 6 as a _preview feature_ because the HTTP/3 specification is not finalized and behavioral or performance issues may exist in HTTP/3 with .NET 6.
+> HTTP/3 is fully supported since .NET 7 only. In .NET 6, HTTP/3 is available as a _preview feature_ because the HTTP/3 specification was not yet finalized and behavioral or performance issues may exist in HTTP/3 with .NET 6.
```

Comment:
```suggestion
> HTTP/3 is fully supported since .NET 7 only. In .NET 6, HTTP/3 is available as a _preview feature_ because the HTTP/3 specification was not yet finalized and behavioral or performance problems may exist in HTTP/3 with .NET 6.
```